### PR TITLE
COMMERCE-5241 Implement Info Framework for CProduct and AssetCategory

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/layout/display/page/CProductLayoutDisplayPageProvider.java [hunk 1]
```diff
@@ -72,10 +78,26 @@ public String getClassName() {
...
+			long classNameId = _portal.getClassNameId(CProduct.class);
+
+			Group group = _groupLocalService.getGroup(groupId);
+
+			Group companyGroup = _groupLocalService.getCompanyGroup(
+				group.getCompanyId());
+
+			FriendlyURLEntry friendlyURLEntry =
+				_friendlyURLEntryLocalService.fetchFriendlyURLEntry(
+					companyGroup.getGroupId(), classNameId, urlTitle);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/layout/display/page/CProductLayoutDisplayPageProvider.java:90]: We can remove the variable.
```suggestion
			Group group = _groupLocalService.getGroup(groupId);

			Group companyGroup = _groupLocalService.getCompanyGroup(
				group.getCompanyId());

			FriendlyURLEntry friendlyURLEntry =
				_friendlyURLEntryLocalService.fetchFriendlyURLEntry(
					companyGroup.getGroupId(),
					_portal.getClassNameId(CProduct.class), urlTitle);
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by ivicac** [comment]
@alecsloan there are now some conflicting files :(

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
Resent at https://github.com/liferay-commerce/liferay-portal/pull/1495

**Pull request closed without merging**